### ISSUE TITLE
Edit Request for Getting Started Wiki ( "Engine.run is not a function" error)

### ISSUE THREAD

**Issue by sosunnyproject** [liabru/matter-js#999]
Hello, 
I was using Matter JS with p5 JS, following Getting Started wiki, and encountered a `Engine.run is not a function` error message. Then I solved by using `Runner.run(runner, engine)` , referring to this thread https://github.com/liabru/matter-js/issues/809 .

While console.logging, I found out that `Engine` in 0.17 v does not support `run` function anymore. But in [Getting Started Doc](https://github.com/liabru/matter-js/wiki/Getting-started) and  [official API page](https://brm.io/matter-js/docs/classes/Engine.html), they still explain to use Engine.run( ).  

When I imported script of older version, like this `<script src=" https://cdnjs.cloudflare.com/ajax/libs/matter-js/0.14.2/matter.min.js"></script>` ,  `Engine.run(engine)` worked fine (in 0.14 v.)

But when I downloaded the latest (0.17 v) matter.js file, and imported `<script src="matter.js"></script>`, the console shows `Engine.run is not a function` error message.

I think the code example could be changed like below. Thanks!

```js
// module aliases
var Engine = Matter.Engine,
    Render = Matter.Render,
    Bodies = Matter.Bodies,
    Composite = Matter.Composite,
    Runner = Matter.Runner;

// create an engine
var engine = Engine.create();
var runner = Runner.create();

// create a renderer
var render = Render.create({
    element: document.body,
    engine: engine
});

// create two boxes and a ground
var boxA = Bodies.rectangle(400, 200, 80, 80);
var boxB = Bodies.rectangle(450, 50, 80, 80);
var ground = Bodies.rectangle(400, 610, 810, 60, { isStatic: true });

// add all of the bodies to the world
Composite.add(engine.world, [boxA, boxB, ground]);

// run the engine
Runner.run(runner, engine);

// run the renderer
Render.run(render);
```

**Comment by liabru** [comment]
Thanks for raising this! 

The `Engine.run` alias was only meant to be deprecated as of `0.17.0` but not yet removed, so I've just pushed `0.17.1` that restores that alias for now but marks it as deprecated.

I've also updated the wiki to use `Runner.run` instead, thanks again for letting me know it was still there. The wiki is definitely due some further updates too, hopefully I can get around to that soon.
